TypeScript/publish.ts

```diff
@@ -81,20 +90,20 @@ export function publish(
 							console.log(
 								`Found differences in command ${module.name}`
 							);
-							await guildcmd.edit(commandData).catch(c);
 							console.log(
 								`${module.name} updated with new data successfully!`
 							);
+							guildcmd.edit(commandData).catch(c);
```

Comment:
This should be before 93-95 because it has not been updated successfully unless 96 does not throw

Reply:
yeah i thought about that, since they are not awaited, the order will of logs will be fucked up

Reply:
i'll reorder it tho

Reply:
just guild.edit.then(log).catch()